src/pool/mod.rs

```diff
@@ -154,7 +154,6 @@
 //!
 //! [1]: https://static.docs.arm.com/ddi0403/eb/DDI0403E_B_armv7m_arm.pdf
 
-#[cfg(not(armv6m))]
```

Comment:
Note: This is okay to remove, as the entire pool module is removed when the `cas` feature is not set, and it is not possible to set the `cas` feature with armv6m targets.